etc/third_party_vulnerabilities.md

```diff
@@ -0,0 +1,37 @@
+# 3rd Party Dependency Vulnerabilities
+
+This document tracks vulnerabilities in 3rd party dependencies that directly affect the MongoDB C++ Driver standard release product.
+
+> [!IMPORTANT]
+> The "standard release product" is defined as the set of files which are _installed_ by a configuration, build, and install of the MongoDB C++ Driver. This includes static/shared library files, header files, and packaging files for supported build configurations. Vulnerabilities for 3rd party dependencies that are bundled with or required by the standard release product are reported in this document.
```

Comment:
```suggestion
> The "standard release product" is defined as the set of files which are _installed_ by a configuration, build, and install of the MongoDB C++ Driver. This includes static/shared library files, header files, and packaging files for supported build configurations. Vulnerabilities for 3rd party dependencies that are bundled with the standard release product are reported in this document.
```

Suggest removing "or required by". [Scope: SSDLC Policy Conformance for Client Libs](https://docs.google.com/document/d/1mXxDdY38nGVKWJhIbH83X6mcHsxllAiunH2hg-NdDsY/edit) notes:

> For purposes of reporting and vulnerability tracking, third-party dependencies only refers to bundled dependencies that ship with a driver.

Example: I expect boost is excluded due to not being bundled. Yet I expect boost could be considered a required dependency (it is one option to satisfy the required polyfill dependency).